pytorch_object_detection/detection_models/__init__.py

```diff
@@ -17,15 +17,20 @@ class DetectionPipeline(ABC):
     Class defining basic functionality to run pretrained object detection model on image or batch of images.
     """
 
-    def __init__(self):
+    def __init__(self, run_on_gpu: bool = True):
```

Comment:
Is this parameter implemented in child classes?

Reply:
they do not implement init